### PR TITLE
perf(scoop-config): Update feedback for `scoop config <xxx> $null`

### PR DESCRIPTION
**Pull request by yi-Xu-0100**
#### Description

update feedback for `scoop config <xxx> $null`.

#### Checklist:

- [x] I have read the [Contributing Guide](https://github.com/ScoopInstaller/.github/blob/main/.github/CONTRIBUTING.md).
- [x] I have ensured that I am targeting the `develop` branch.
- [x] I have updated the documentation accordingly.
- [ ] I have updated the tests accordingly.
- [ ] I have added an entry in the CHANGELOG.

### CODE REVIEW

**Comment by niheaven** [comment]
`set_config` should remove a value if being set to `$null`, not `scoop config <xxx> $null`. You may misunderstood the test and `scoop config`.

**Comment by yi-Xu-0100** [comment]
@niheaven I try to use `scoop config name $null`, but it will return value of name when the name has value. 😵‍💫

I think that the response is confused for user and need to give feedback, then I find the `set_config` can deal with the `$null`, so I think the scoop config should deal with it also.🤔

**Comment by niheaven** [comment]
Since `scoop config <xxx> $null` is the same as `scoop config <xxx>`, the `$value` will be `$null` if ignored, so the command just output the config value.

In this PR, all command that may show option value will remove the option, such as:

image

**Comment by yi-Xu-0100** [comment]
@niheaven I understand the reason for `scoop config name $null` not working. I update the feedback for it to notification the right way to remove.

File: libexec/scoop-config.ps1 [hunk 1]
```diff
@@ -147,7 +147,12 @@ if (!$name) {
     if($null -eq $value) {
         Write-Host "'$name' is not set"
     } else {
-        $value
+        # Since `scoop config <xxx> $null` is the same as `scoop config <xxx>`,
+        # so it only can remove <xxx> with `scoop config rm <xxx>`.
+        # https://github.com/ScoopInstaller/Scoop/pull/5090#issuecomment-1207389924
+        Write-Host "'$name' is '$value'."
```
Comment (rashil2000) [inline libexec/scoop-config.ps1:153]: Just a note: Using Write-Host here will prevent users from storing config values, e.g. in a script like this:
```pwsh
$updated_at = scoop config lastupdate
```

**Comment by yi-Xu-0100** [comment]
@niheaven @rashil2000 I have no idea to notify, maybe only me need this. Thanks for all your review.

**Pull request closed without merging**